### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by bencmbrook**
Updated flow of docs + nits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,234 +1,114 @@
...
+
+    customer_ingress:
+      enabled: true
+      className: alb
+      annotations:
+        alb.ingress.kubernetes.io/certificate-arn: <CERT_ARN>
+        alb.ingress.kubernetes.io/healthcheck-path: /health
+        alb.ingress.kubernetes.io/healthcheck-port: '5039'
+        alb.ingress.kubernetes.io/healthcheck-protocol: HTTP
+        alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS": 5039}]'
```
Comment (dipack95) [inline README.md:58]: Shouldn't the listener ports be HTTP for HTTP configurations?

File: README.md [hunk 2]
```diff
@@ -513,7 +369,7 @@ customer_ingress:
     alb.ingress.kubernetes.io/certificate-arn: <CERT_ARN>
     alb.ingress.kubernetes.io/healthcheck-path: /health
     alb.ingress.kubernetes.io/healthcheck-protocol: HTTP
-    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS": 5039}]'
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS": 5039}, {"HTTPS": 3030}]'
```
Comment (dipack95) [inline README.md:372]: This seems incorrect? The ALB for Sombra does not listen for the Pathfinder service too, right?

File: README.md [hunk 3]
```diff
@@ -527,17 +383,162 @@ envs:
...
+| DD_SERVICE_NAME                        | no                                                                             | `transcend-hosted-sombra`                                                                                                  | no     | The name for your Sombra. Only applicable when you are using Datadog agent for log and metrics collection.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```
Comment (dipack95) [inline README.md:492]: Do we need to expose these DD service variables?

File: README.md [hunk 4]
```diff
@@ -527,17 +383,162 @@ envs:
...
+| LLM_SERVER_PORT        | no       | 6081             | no     | Port on which server listen to.                                                                                      |
+| LLM_SERVER_CONCURRENCY | no       | (cpu count) \* 2 | no     | The number of worker processes for handling requests.                                                                |
+| LLM_SERVER_TIMEOUT     | no       | 120              | no     | Workers silent for more than this many seconds are killed and restarted.                                             |
+| DD_SERVICE             | no       | N/A              | no     | Service name. Only relevant for log and metrics collection through Datadog agent.                                    |
+| DD_ENV                 | no       | N/A              | no     | Deployment environment. Only relevant for log and metrics collection through Datadog agent.                          |
+| DD_AGENT_HOST          | no       | N/A              | no     | Host IP to which logs and metrics are forwarded. Only relevant for log and metrics collection through Datadog agent. |
```
Comment (dipack95) [inline README.md:506]: Same comment here

**Review by dipack95 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (bencmbrook) [reply README.md:372]: Will defer to @shashiranjan84 on that.

I reordered the examples, so it looks diffed, but I didn't actually change the config

**Comment by bencmbrook** [comment]
@dipack95 we should action these comments, but since the change here is just a bunch of reordering, the diff is hard to track. The env var tables and the YAML snippets are unchanged. 

I'll merge this and then we should reopen a new PR for those

File: README.md (hunk 1 above)
Comment (shashiranjan84) [reply README.md:58]: `listen-ports` here defining the ports on which Load balancer will listen which we mostly use HTTPS  to keep traffic b/w client and load balancer over TLS

Comment (dipack95) [reply README.md:58]: Nice, thanks for the explanation!

**Pull request merged**